src/style.css

```diff
@@ -1,3 +1,64 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: black;
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  width: 1200px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+  padding: 0 50px;
+}
+
+.photo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.nav__item {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  margin-left: 20px;
+  padding: 7px 0;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
```

Comment:
This works however it can be done in a better, more elegant way I'd say.

Take a look at the last point of the checklist:
_Selector :not(:first-child) has been used while setting indents in navigation_

Don't set the maring in the `nav__item`. Instead do it in a new selecor with `:not(:first-child)` pseudo-classes